Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,25 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250])
+    var
+        ConstUpgradeTags: Record "Upgrade Tags";
+    begin
+        ConstUpgradeTags.Reset;
```

Comment:
Naming - I would drop Const, it does not bring value. UpgradeTags is good enough, alternative is ExistingUpgradeTags.

Reply:
removed Const